commercetools-models/src/test/java/io/sphere/sdk/shoppinglists/commands/ShoppingListUpdateCommandIntegrationTest.java

```diff
@@ -3,6 +3,7 @@
 import io.sphere.sdk.models.LocalizedString;
 import io.sphere.sdk.shoppinglists.*;
 import io.sphere.sdk.shoppinglists.commands.updateactions.*;
+import io.sphere.sdk.stores.StoreFixtures;
```

Comment:
seems you missed a test with a store ;)

Reply:
I just saw that if the method has been automatically generated with the annotation @HasUpdateAction there are no tests related like if you see the getAnonymousId() ;)